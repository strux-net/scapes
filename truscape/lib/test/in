style
  var3

  # a comment
  var4
	4

	5
  var5[]
	left
	      right
