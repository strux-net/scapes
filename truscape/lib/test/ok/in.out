$var1	1
$var2	2

2.1
2.2
$var3	
@B	:north south  east west:
$#B	4
$B	
$var4	4

5
@var5	left       right
